sdk-actors/src/test/java/io/dapr/actors/runtime/ActorRuntimeConfigIT.java

```diff
@@ -0,0 +1,98 @@
+package io.dapr.actors.runtime;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.actors.runtime.ActorRuntimeConfig;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ActorRuntimeConfigIT extends ActorRuntimeConfig {
+
+  private String actorIdleTimeout;
+
+  private String actorScanInterval;
+
+  private String drainOngoingCallTimeout;
+
+  private Boolean drainBalancedActors;
+
+  private Collection<String> entities = new ArrayList<>();
+
+  public ActorRuntimeConfigIT(String actorIdleTimeout, String actorScanInterval, String drainOngoingCallTimeout, Boolean drainBalancedActors) {
```

Comment:
It might be confusing having "IT" in the file and class name, because that's the filter that IT's use (even though there's not tests inside).  Maybe just add "My" or "Test" to ActorRunTimeConfig instead of "IT".

Reply:
oh forgot to delete that class, is not really needed.